internal/jasmine_node_test/jasmine_node_test.bzl

```diff
@@ -56,10 +56,11 @@ def jasmine_node_test(
         tags = tags,
     )
 
+    jasmine_runner_label = Label("//internal/jasmine_node_test:jasmine_runner.js")
+
     all_data = data + srcs + deps
-    all_data += [Label("//internal/jasmine_node_test:jasmine_runner.js")]
     all_data += [":%s_devmode_srcs.MF" % name]
-    entry_point = "build_bazel_rules_nodejs/internal/jasmine_node_test/jasmine_runner.js"
+    entry_point = jasmine_runner_label.relative(":jasmine_runner.js")
```

Comment:
this is surprising, why does it need a relative calculation when the jasmine_runner_label already points to the right file?

Reply:
yup. there is no reason for that 🤷‍♂